docs/optimization/summary.md

```diff
@@ -0,0 +1,13 @@
+# Performance
```

Comment:
For me and seeing the content that section for me is not usefull and just moving Optimization under Guides should be enough for my point of view. We discuss that this morning so if you are happy with that I'm too.

Reply:
I would disagree with @BourgerieQuentin : it gives the main lines + it gives a bit of options the user should try (eg, enabling data parallelism)

Reply:
We agreed in our discussions, so resolving.